homeassistant/components/plugwise/climate.py

```diff
@@ -145,17 +145,12 @@ def preset_mode(self) -> str | None:
         """Return the current preset mode."""
         return self.device.get("active_preset")
 
-    @property
-    def extra_state_attributes(self) -> Mapping[str, Any] | None:
-        """Return entity specific state attributes."""
-        return {
-            "available_schemas": self.device["available_schedules"],
-            "selected_schema": self.device["select_schedule"],
-        }
-
     @plugwise_command
     async def async_set_temperature(self, **kwargs: Any) -> None:
         """Set new target temperature."""
+        if ATTR_HVAC_MODE in kwargs:
```

Comment:
I know it's two small things but they should be in separate PR's anyhow as they are not related to each other.
Move this improvement into it's own PR and remove it from here

Reply:
Done.